protocol/x/perpetuals/client/cli/query_premiums.go

```diff
@@ -0,0 +1,55 @@
+package cli
+
+import (
+	"context"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/dydxprotocol/v4-chain/protocol/x/perpetuals/types"
+	"github.com/spf13/cobra"
+)
+
+func CmdQueryPremiumSamples() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "get-premium-samples",
+		Short: "get the PremiumSamples",
```

Comment:
Could you update this to "Get PremiumSamples from the current `funding-tick` epoch"